docs/src/tutorials/device-monitoring.md

```diff
@@ -0,0 +1,119 @@
+   * Thin-edge.io expects the collectd metrics to be published on the local MQTT bus.
+     Hence, you must enable the [MQTT write plugin of collectd](https://collectd.org/documentation/manpages/collectd.conf.5.shtml#plugin_mqtt).
+   * The MQTT plugin is available on most distribution of `collectd`, but this is the case on MacOS using homebrew.
+     If you are missing the MQTT plugin, please recompile `collectd` to include the MQTT plugin.
+     See [https://github.com/collectd/collectd](https://github.com/collectd/collectd) for details.
+2. __RRDTool and CSV should be disabled__
+   * `#LoadPlugin rrdtool`
+   * `#LoadPlugin csv`
+   * The risk with these plugins is to run out of disk space on a small device.
+   * With thin-edge.io the metrics collected by `collectd` are forwarded to the cloud,
+     hence it makes sense to [disable Local storage](https://github.com/collectd/collectd/issues/2668).
+3. __Cherry-pick the collected metrics__
+   * `Collectd` can collect a lot of detailed metrics,
+      and it doesn't always make sense to forward all these data to the cloud.
+   * See the [`tedge-collectd.conf` example](https://github.com/thin-edge/thin-edge.io/blob/main/configuration/contrib/collectd/tedge-collectd.conf)
+     to see how to use the `match_regex` plugin to select the metrics of interest.
+     
+## Enable thin-edge monitoring
+
+To enable monitoring on your device, you have now to launch the `collectd-mapper` daemon process.
+
+``` shell
+ sudo systemctl start tedge-collectd-mapper
+```
+
+This process subscribes to the `collectd/#` topics and emits translated measurements to the `tedge/measurements` topic.
+You can inspect the collected and translated metrics, by subscribing to these topics:
+
+The metrics collected by `collectd` are emitted to subtopics named after the collectd plugin and the metric name:
+
+```
+$ tedge mqtt sub 'collectd/#'
+
+[collectd/raspberrypi/cpu/percent-active] 1623076679.154:0.50125313283208
+[collectd/raspberrypi/memory/percent-used] 1623076679.159:1.10760866126707
+[collectd/raspberrypi/cpu/percent-active] 1623076680.154:0
+[collectd/raspberrypi/df-root/percent_bytes-used] 1623076680.158:71.3109359741211
+[collectd/raspberrypi/memory/percent-used] 1623076680.159:1.10760866126707
+
+```
+
+The `tedge-collectd-mapper` translates into the [thin-edge.io JSON](../architecture/thin-edge-json.md) format,
+[grouping the measurements](../references/bridged-topics.md#collectd-topics) emitted by each plugin:
+
+```
+tedge mqtt sub 'tedge/measurements'
+
+[tedge/measurements] {"time":"2021-06-07T15:38:59.154895598+01:00","cpu":{"percent-active":0.50251256281407},"memory":{"percent-used":1.11893578135189}}
+[tedge/measurements] {"time":"2021-06-07T15:39:00.154967388+01:00","cpu":{"percent-active":0},"df-root":{"percent_bytes-used":71.3110656738281},"memory":{"percent-used":1.12107875001658}}
+```
+
+From there, if the device is actually connected, these monitoring metrics will be forwarded to the cloud.
```

Comment:
```suggestion
From there, if the device is actually connected to a cloud platform like Cumulocity, these monitoring metrics will be forwarded to the cloud.
```

Reply:
That suggested change since the below samples are for Cumulocity.

Reply:
Changed